web/msg/testdata/send.json

```diff
@@ -62,6 +62,7 @@
             "urn": "tel:+16055741111?id=10000",
             "text": "hello",
             "attachments": [],
+            "quick_replies": null,
```

Comment:
Is having `null` here fine  or we should have `[]`? @rowanseymour

Reply:
I think make sure it works when omitted - but we shouldn't be sending `null` for a slice

Reply:
That is the Msg serialization here,

I think we can make `QuickReplies()` return an empty list always as https://github.com/nyaruka/mailroom/blob/5b1294dbadd4e5cf7721dcba74409ad5a61007b9/core/models/msgs.go#L232

Reply:
And I guess we the change above for Quick Replies we need to do that

Reply:
Ah this is the response, I thought I was looking at a null in the request.. yeah we shouldn't return null for a list here